Spanner/tests/System/PgWriteTest.php

```diff
@@ -885,11 +884,10 @@ public function testExecuteUpdateBatchSingleStatement()
      * Run batchUpdate with no statement, expect an error to be returned, with
      * an empty result list.
      *
-     * @group spanner-write-batch-dml
```

Comment:
Should this annotation have been dropped?

Reply:
nope